lib/bitstyles_phoenix.ex

```diff
@@ -33,4 +33,22 @@ defmodule BitstylesPhoenix do
   showcase("Inline errors", """
     ui_error_tag("Some error")
   """)
+
+  showcase_section("Flash", "Components.Flash.ui_flash/2")
+
+  showcase("Brand-1", """
+    ui_flash("Something you may be interested to hear", variant: "brand-1")
+  """)
+
+  showcase("Positive", """
+    ui_flash("Changes succesfully saved", variant: "positive")
+  """)
+
+  showcase("Warning", """
+    ui_flash("Login not successful, please try that again", variant: "warning")
+  """)
+
+  showcase("Danger", """
+    ui_flash("An error occured, please contact an admin", variant: "danger")
+  """)
```

Comment:
We should add one unit test and showcase for the `do` option like this (i think): 

```
ui_flash("An error occured, please contact an admin", variant: "danger") do
  content_tag(:span, "test")
end
```

Reply:
Yeh I never got the to work (would be great for buttons too!). Sadly I can’t get this to work in the showcase nor the doctests 🤔 Probably I’m just getting the nesting/syntax wrong…

Reply:
Added :)